gui/src-tauri/src/state.rs

```diff
@@ -78,33 +91,20 @@ impl WindowState {
 	}
 }
 
-pub trait WindowBuilderExt {
-	fn restore_state(self, state: &WindowState) -> Self;
-}
-
-impl WindowBuilderExt for tauri::WindowBuilder<'_> {
-	fn restore_state(self, state: &WindowState) -> Self {
-		if !state.is_old() {
-			return self;
-		}
-		self.inner_size(state.width, state.height)
-			// .maximized(state.maximized)
-	}
-}
-
-trait MonitorExt {
+pub trait MonitorExt {
```

Comment:
If you are confident this is easier to read for a rust beginner you could keep the Ext trait, but I still think its easier for them to not use this pattern and just have a regular function

Reply:
thank you ><